clickhouse-admin/src/clickhouse_cli.rs

```diff
@@ -161,6 +168,25 @@ impl ClickhouseCli {
         .await
     }
 
+    pub async fn system_timeseries_avg(
+        &self,
+        settings: SystemTimeSeriesSettings,
+    ) -> Result<Vec<SystemTimeSeries>, ClickhouseCliError> {
+        let log = self.log.clone().unwrap();
+        let query = settings.query_avg();
+
+        info!(&log, "Querying system database"; "query" => &query);
```

Comment:
Does this need to be an `info` log? What about a `debug` or `trace` log instead?